src/amp.js

```diff
@@ -91,6 +91,7 @@ if (shouldMainBootstrapRun) {
   startupChunk(self.document, function initial() {
     /** @const {!./service/ampdoc-impl.AmpDoc} */
     const ampdoc = ampdocService.getAmpDoc(self.document);
+    installPlatformService(self);
```

Comment:
@jridgewell Can we get your approval on the change here. Thanks

Performance Service needs Platform service because `FIRST_CONTENTFUL_PAINT` only works in Chrome and Opera.

Reply:
LGTM, platform's constructor is simple enough.